### PR TITLE
Cut the 2.14.0-rc.0 release

### PR DESCRIPTION
**Pull request by csmarchbanks**
Cut the first 2.14.0 release candidate. It is possible I will add one more PR to this release, but I want to get any comments sooner than later.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+## 2.14.0-rc.0 / 2019-11-06
+
+* [FEATURE] API: `/api/v1/status/runtimeinfo` and `/api/v1/status/buildinfo` endpoints added for use by the React UI. #6243
+* [FEATURE] React UI: implement the new experimental React based UI. #5694 and many more
+  * Can be found by under `/new`.
+  * Not all pages are implemented yet.
+* [FEATURE] Status: Cardinality statistics added to the Runtime & Build Information page. #6125
+* [ENHANCEMENT] Remote write: fix delays in remote write after a compaction. #6021
+* [ENHANCEMENT] Service discovery: errors are no longer logged on context cancellation. #6116, #6133
```
Comment (brian-brazil) [inline CHANGELOG.md]: Bugfix?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+* [ENHANCEMENT] UI: Alerts can be filtered by state. #5758
+* [BUGFIX] Build: Fix Solaris build. #6149
+* [BUGFIX] Promtool: Remove false duplicate rule warnings when checking rule files with alerts. #6270
+* [BUGFIX] Remote write: restore use of deduplicating logger in remote write. #6113
+* [BUGFIX] Remote write: do not reshard when unable to send samples. #6111
+* [BUGFIX] Service discovery: fix custom-sd adapter on windows. #6217
```
Comment (brian-brazil) [inline CHANGELOG.md]: This isn't part of the prometheus release

**Comment by csmarchbanks** [comment]
Thanks for the review, I have updated this for the comments, and rebased to include #6279. Right now I have 6279 listed as security/bugfix, is that ok or should it just be bugfix?

**Comment by brian-brazil** [comment]
:+1: 

security/bugfix and on top is right, as it's basically a reflected XSS.

**Comment by csmarchbanks** [comment]
/benchmark cancel

Benchmark looks good.

**Pull request merged**